r/R/table.R

```diff
@@ -184,3 +184,6 @@ Table$create <- function(..., schema = NULL) {
 
 #' @export
 names.Table <- function(x) x$ColumnNames()
+
+#' @export
+length.ArrowTabular <- function(x) x$num_columns
```

Comment:
Could you please move this to `arrow-tabular.R`? Thanks!

Reply:
Done!